crypto/ct/ct_vfy.c

```diff
@@ -219,7 +219,7 @@ int SCT_verify(const SCT_CTX *sctx, const SCT *sct)
     if (sct->log_id_len != sctx->pkeyhashlen ||
         memcmp(sct->log_id, sctx->pkeyhash, sctx->pkeyhashlen) != 0) {
         CTerr(CT_F_SCT_VERIFY, CT_R_SCT_LOG_ID_MISMATCH);
-        return 0;
+        return -1;
```

Comment:
And this, too, changes the contract, and needs to be documented.

In general, there is no point returning different values if the caller cannot do anything about them programmatically. For debugging, we have the error stack.